### DISCUSSION TITLE
May I use the file 'english_10k.json' in my student project

### DISCUSSION THREAD

**Discussion by needlesslygrim** [monkeytypegame/monkeytype#3716] [Questions] [answered]
I am working on a project for school which requires me to generate a passphrase, so I thought I could use the list of top 10,000 most popular English words from Monkeytype. [This](https://github.com/monkeytypegame/monkeytype/blob/53e890272405e95b88695ff2128696db31038ad0/frontend/static/languages/english_10k.json) is the file I wanted to use. I will make the project public on my GitHub account when it's finished and probably licence it under GPLV3 as that's what this project is licensed as. Is there any issue with me using this JSON file? Thanks in advance.

**Comment by Miodec** [comment, marked as answer]
No issues - go for it.

**Reply by needlesslygrim** [reply]
Thanks, that's really helpful
